src/rules/sort-keys.js

```diff
@@ -0,0 +1,150 @@
+/**
+ * @fileoverview Rule to require JSON object keys to be sorted. Cribbed largely from https://github.com/eslint/eslint/blob/main/lib/rules/sort-keys.js
+ * @author Robin Thomas
+ */
```

Comment:
For clarity to non-native English speakers
```suggestion
/**
 * @fileoverview Rule to require JSON object keys to be sorted. Copied largely from https://github.com/eslint/eslint/blob/main/lib/rules/sort-keys.js
 * @author Robin Thomas
 */
```